presto-hive/src/main/java/com/facebook/presto/hive/security/LegacyAccessControl.java

```diff
@@ -101,7 +101,8 @@ public void checkCanDropTable(ConnectorTransactionHandle transaction, ConnectorI
         }
 
         TransactionalMetadata metadata = hiveTransactionManager.get(transaction);
-        MetastoreContext metastoreContext = new MetastoreContext(identity, context.getQueryId().getId(), context.getClientInfo(), context.getSource());
+        // TODO: Does it impact to keep it empty map?
```

Comment:
I am not sure, so asking this question. AccessControlContext does have subset of ConnectorSession fields (used for permission/access control (it seems).